pkg/web/routes.go

```diff
@@ -195,6 +198,7 @@ func (s *Server) setupRoutes() (err error) {
 			users.GET("/:id", authorize(permiss.UsersView), s.UserDetail)
 			users.PUT("/:id", authorize(permiss.UsersManage), s.UpdateUser)
 			users.DELETE("/:id", authorize(permiss.UsersManage), s.DeleteUser)
+			users.POST("/:id/password", authorize(permiss.UsersManage), s.ChangeUserPassword)
```

Comment:
This is the API-based user management change password. Any user's password can be changed by anyone with the UsersManage permission (Admin role)